hooks/publish.py

```diff
@@ -354,7 +354,7 @@ def validate(self, task_settings, item):
             )
             return False
 
-        if item.properties.get("is_sequence"):
+        if item.properties.get("is_sequence") and item.properties.get("path"):
```

Comment:
Feels like this entire block down to line 397 should be wrapped in the `if item.properties.get("path")` conditional.

Reply:
I don't think so. `path` is simply the work file path. Even if that doesn't exist (eg. for mari textures), we still need to set the publish path (else block, lines 360-362) and confirm there are no conflicting publishes (rest of the code till 397)